src/docs/tutorial-branding/configuration/index.md

```diff
@@ -130,6 +115,21 @@ other variables:
 
 </extrainfo>
 
+Another configuration defined in configDefault.js is **siteTitle** - and
+then that variable is used in `<meta>` tags and
+[webpage's schema](https://schema.org/).
+
+```js
+// Site title is needed in meta tags (bots and social media sharing reads those)
+const siteTitle = 'Biketribe';
+```
+
+### Task: _Set listing's minimum price_
```

Comment:
This is set in Console, and if the value exists in both Console/asset and locally, the asset value gets used. Should we omit the listing minimum price setting, or describe how to set it in Console?

Reply:
I think we just replace that with a picture of Console accompanied by a text that you can change the listing price in Console. I can handle that. @SariSaar